tonic/src/transport/channel/mod.rs

```diff
@@ -9,24 +9,26 @@ pub use endpoint::Endpoint;
 #[cfg(feature = "tls")]
 pub use tls::ClientTlsConfig;
 
-use super::service::{Connection, ServiceList};
+use super::service::{Connection, DynamicServiceStream};
 use crate::{body::BoxBody, client::GrpcService};
 use bytes::Bytes;
 use http::{
     uri::{InvalidUri, Uri},
     Request, Response,
 };
 use hyper::client::connect::Connection as HyperConnection;
+use std::hash::Hash;
```

Comment:
minor nit: can we move this import into the one below?

Reply:
done